modules/phase_field/src/ics/BimodalSuperellipsoidsIC.C

```diff
@@ -124,9 +123,8 @@ BimodalSuperellipsoidsIC::computeSuperellipsoidExponents()
     _ns[i] = _small_n;
 }
 
-
 void
-BimodalSuperellipsoidsIC::computeSuperellipsoidCenters()
+BimodalSuperellipsoidsIC::computeCircleCenters()
```

Comment:
It is a wip commit. I'm still working on this.

On Thu, Jul 7, 2016, 6:19 PM laagesen <notifications@github.com> wrote:

> In modules/phase_field/src/ics/BimodalSuperellipsoidsIC.C
> <https://github.com/idaholab/moose/pull/7297#discussion_r70009841>:
>
> >  void
> > -BimodalSuperellipsoidsIC::computeSuperellipsoidCenters()
> > +BimodalSuperellipsoidsIC::computeCircleCenters()
>
> why the change to computeCircleCenters?
>
> —
> You are receiving this because you authored the thread.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7297/files/7bfa4b1a528c67b3a24f7500e55d7100030650b6#r70009841>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMWPkLGrfmy3BPf1uC2aioub6DfkAStks5qTZeMgaJpZM4JHfg->
> .
>